pkg/sdk/logging/model/syslogng/config/render/helpers.go

```diff
@@ -132,6 +133,20 @@ func Literal[T LiteralTypes](v T) Renderer {
 	}
 }
 
+// ArrowMap renders a map as a key=>value style map used at various places of the config grammar
+func ArrowMap(v map[string]string) Renderer {
+	keys := []string{}
+	for key := range v {
+		keys = append(keys, key)
+	}
```

Comment:
```suggestion
	keys := maps.Keys(v)
```
using `golang.org/x/exp/maps`